lib/cmds/merge_cmds/export.ts

```diff
@@ -142,17 +152,21 @@ const exportEnvironmentMigration = async ({
       targetEnvironmentId,
       spaceId: activeSpaceId
     })
+
+    if (!migration) {
+      throw new Error(mergeErrors['MigrationCouldNotBeExported'])
+    }
   } catch (e) {
     if (e instanceof Error) {
       throw e.message
     }
 
-    throw new Error('Migration could not be exported.')
+    throw new Error(mergeErrors['MigrationCouldNotBeExported'])
```

Comment:
All of this is already handled by callExportAppAction (we catch and rethrow with new error messages there), so I think here we can just propagate the errors, no?